app/controllers/catalog_controller.rb

```diff
@@ -268,7 +268,7 @@ class CatalogController < ApplicationController
     # or can be specified manually to be different.
 
     keyword_fields = [
-      'title_precise_tesim^100', 'author_tesim^50', 'author_addl_display_tesim^50',
+      'title_precise_tesim^100', 'author_tesim^50', 'author_addl_display_tesim^50', 'barcode_ssim',
```

Comment:
Since this is going into search fields, can you add the field into `spec/system/targeted_field_search_spec.rb` in both `:fields` L#5 and the test on L#69?